cfme/networks/views.py

```diff
@@ -82,11 +82,7 @@ class NetworkProviderDetailsView(BaseLoggedInPage):
     @View.nested
     class entities(View):  # noqa
         """ Represents details page when it's switched to Summary/Table view """
-        properties = SummaryTable(title="Properties")
```

Comment:
**Required**: Please update deleted properties usages in tests to new summary property